provider/kubernetes/kubernetes.go

```diff
@@ -294,6 +296,25 @@ func (p *Provider) loadIngresses(k8sClient Client) (*types.Configuration, error)
 	return &templateObjects, nil
 }
 
+func getRuleForPath(pa v1beta1.HTTPIngressPath, i *v1beta1.Ingress) string {
+	if len(pa.Path) > 0 {
```

Comment:
Can we check for `== 0` and return an empty string right away? That saves us a level of indentation and keeps the value returned in case of an empty path closer to its detection.